napari/utils/colormaps/colormap.py

```diff
@@ -161,13 +161,12 @@ class LabelColormap(Colormap):
     interpolation: ColormapInterpolationMode = ColormapInterpolationMode.ZERO
 
     def map(self, values):
-        from napari.utils.colormaps.colormap_utils import low_discrepancy_image
+        values = np.atleast_1d(values)
+        values[values == None] = 0  # noqa: E711
```

Comment:
Should this `0` actually be the `background_label` instead of hardcoded, or is it unrelated?

Reply:
hm. Maybe it should. I do not understand why we need to support `None`. Did you know?

Reply:
I don't remember exactly, but there's several places where we have both this `None` label and the background label special-cased. It might be related to NaNs? Going through the `Labels` I'm not sure it's needed actually...

Reply:
It seems some of this None-related code was worked on in https://github.com/napari/napari/pull/3275; @DragaDoncila, what are we missing here?

Reply:
@DragaDoncila friendly reminder :)

Regardless of what happens to the `None` case, I think the `0` should be `self.background_label`, which should be added to the colormap.

Reply:
I have very little memory of this, but I do remember there was a legit reason for it. The only thing I can think of is possible `None` was the color used for labels that did not have an assigned color in direct mode. I also think that 0 up there should almost definitely be `background_label`. Will see if I can suss what exactly the `None` was used for - currently on `main` it doesn't seem to do anything - certainly, it doesn't color the unmapped labels.

Reply:
@brisvag @Czaki that is indeed what it was designed to do. See original [comment](https://github.com/napari/napari/pull/1339#pullrequestreview-425861557). Thanks @jni for hunting it down! We at a minimum would want to document this, if we're going to fix it, but maybe we should actually implement it in a clearer way - @jni suggested maybe a `default_color` property?